app/views/staff/events/show.html.haml

```diff
@@ -103,24 +117,26 @@
               %tr
                 %td.text-primary
                   %strong CFP Closes Date:
-                - if event.closes_at.present?
+                - if event.closes_at && (event.closes_at > Time.current)
```

Comment:
One thing catches my eye here: there no timezone being taken into account in the comparison. I'll review this a bit and follow up.